app/assets/stylesheets/tinymce/skins/alchemy/skin.min.css.scss

```diff
@@ -616,12 +618,12 @@ body .mce-abs-layout-item,.mce-abs-end {
   display: inline-block;
   *display: inline;
   *zoom: 1;
-  @include box-shadow(inset 0 1px 0 rgba(255,255,255,0.2),0 1px 2px rgba(0,0,0,0.05));
-  @include background(linear-gradient(color-stops($sb-base-color, darken($sb-base-color, 10%))));
+  box-shadow: inset 0 1px 0 rgba(255,255,255,0.2),0 1px 2px rgba(0,0,0,0.05);
+  @include background(linear-gradient(color-stops($white, darken($white, 10%))));
```

Comment:
The syntax is actually without the `color-stops`:

```sass
@include background(linear-gradient($white, darken($white, 10%)));
```

see: http://bourbon.io/docs/#background